src/frontend/src/flows/addDevice/welcomeView/registerTentativeDevice.ts

```diff
@@ -96,13 +96,15 @@ export const registerTentativeDevice = async (
 
   // If everything went well we can now ask the user to authenticate on an existing device
   // and enter a verification code
-  return await showVerificationCode(
+  await showVerificationCode(
```

Comment:
this returns `"ok"` which was previously returned; that ensured that if `showVerificationCode` was ever modified to return something else (like `"ok" | "failed") the typecheck would warn us here.

Do you think would make sense to ensure the result is `"ok"` here?

Reply:
Not sure but no strong opinion. What would be the check and what should happens if not matching?